packages/ui/cypress/support/page-objects/landingPage.ts

```diff
@@ -13,6 +13,7 @@ export const landingPage = {
   noAccountFoundError: () => cy.get('[data-cy=label-no-account-found]', { timeout: 10000 }),
   linkedAddressNotFound: () => cy.get('[data-cy=label-linked-address-not-found]'),
   resetLinkedAddressButton: () => cy.get('[data-cy=button-reset-linked-address]'),
+  firstMultisigCreationText: () => cy.get('[data-cy=text-first-multisig-creation'),
```

Comment:
Very minor conventions point but we usually use "label" instead of "text" in the page objects / locators (checking text of label reads better in test than checking the text of text)